src/assets/sass/protocol/components/_notification-bar.scss

```diff
@@ -0,0 +1,101 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../includes/lib";
+
+.mzp-c-notification-bar {
+    @include text-display-xxs;
+    background-color: $color-gray-20;
+    border-radius: 4px;
+    border: 1px;
+    box-shadow: 1px 1px 1px rgba(27,31,35,.1);
+    margin: $layout-xs auto 0;
+    height: 42px;
+    font-weight: normal;
+    max-width: $content-sm;
+    text-align: center;
+
+    &.mzp-is-sticky {
+        position: fixed;
+        top: 0;
+        z-index: 2;
+        margin-left: auto;
+        margin-right: auto;
+        left: 0;
+        right: 0;
+    }
+
+    .mzp-c-notification-bar-button {
+        @include image-replaced;
+        @include bidi((
+                (float, right, left),
+                (border-radius, 0 4px 4px 0, 4px 0 0 4px),
+            ));
+        background: $color-gray-40
+            url("#{$image-path}/icons/ui/close-black.svg") center center
+            no-repeat;
+        border: none;
+        display: block;
+        height: 42px;
+        min-width: 0;
+        padding: 0;
+        width: 42px;
+
+        &:hover,
+        &:focus {
+            cursor: pointer;
+        }
+
+        &:focus {
+            outline: 1px dotted $color-white;
```

Comment:
This focus state is pretty low contrast and hard to see compared to browser defaults?